Documentation/01 Getting Started.md

```diff
@@ -65,9 +65,9 @@ Count.main()
 
 In the code above, the `inputFile` and `outputFile` properties use the `@Argument` property wrapper. `ArgumentParser` uses this wrapper to denote a positional command-line input — because `inputFile` is specified first in the `Count` type, it's the first value read from the command-line, and `outputFile` is read second.
 
-We've implemented the command's logic in its `run()` method. Here, we're printing out a message confirming the names of the files the user gave. (You can find a full implementation of the completed command at the end of this guide.)
+We have implemented the command's logic in its `run()` method. Here, we are printing out a message confirming the names of the files the user gave. (You can find a full implementation of the completed command at the end of this guide.)
```

Comment:
The contractions are okay as is. Could you revert those changes?

Reply:
Reverted in f086616 🙂